client/app/telecom/telephony/billingAccount/billing/repaymentHistory/telecom-telephony-billing-account-billing-repayment-history.controller.js

```diff
@@ -38,25 +27,12 @@ angular.module("managerApp").controller("TelecomTelephonyBillingAccountBillingRe
                     consumption.priceValue = consumption.price ? consumption.price.value : null;
                 });
             });
+        }, function (err) {
+            self.consumptionData = [];
+            return new ToastError([$translate.instant("telephony_group_billing_repayment_history_download_error"), (err.data && err.data.message) || ""].join(" "));
```

Comment:
Be careful !!! This is not how `ToastError` works !!!
Here is the declaration : 
https://github.com/ovh-ux/ovh-manager-telecom/blob/3d1d22ba1564d43fa263f9758584513cd7dd4142/client/components/toast-error/toast-error.service.js#L1-L24
It takes as arguments an error object AND a translation key, not a string !!!
If you want a beautiful error message let like it was done before and simply use `Toast.error()` method and reject the error.